pkg/serializer/serializer_test.go

```diff
@@ -199,7 +199,7 @@ func TestSendV1ServiceChecks(t *testing.T) {
 	f.On("SubmitV1CheckRuns", jsonPayloads, jsonExtraHeadersWithCompression).Return(nil).Times(1)
 
 	s := NewSerializer(f)
-
+	s.enableServiceChecksJSONStream = false
```

Comment:
Can you align this with https://github.com/DataDog/datadog-agent/blob/79b8333cb498b6d8c7b26cf8f89eeec15b9a9b9f/pkg/serializer/serializer_test.go#L236-L237
Either way is fine for me